src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -150,13 +150,28 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
     
     /**
      * Updates the feedback question identified by `{@code newAttributes.getId()} 
+     * without keeping the old timestamp
      * For the remaining parameters, the existing value is preserved 
      *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * 
      * Preconditions: <br>
      * * {@code newAttributes.getId()} is non-null and
      *  correspond to an existing feedback question. <br>
      */
     public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes) throws InvalidParametersException, EntityDoesNotExistException {
+        updateFeedbackQuestion(newAttributes, false);
+    }
+    
+    /**
+     * Updates the feedback question identified by `{@code newAttributes.getId()} 
+     * For the remaining parameters, the existing value is preserved 
+     *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * The timestamp is preserved or not based on the value of {@code keepUpdateTimestamp} 
```

Comment:
The update timestamp is changed when this method is called (since it is technically updated) and it depends on when the method is called (and the <code>keepUpdateTimestamp</code> value). I believe it is independent of the <code>newAttributes</code>.